code/controllers/subsystem/ticker.dm

```diff
@@ -114,18 +114,20 @@ SUBSYSTEM_DEF(ticker)
 			if(byond_sound_formats[ext])
 				continue
 		music -= S
-				
+
 	if(isemptylist(music))
 		music = world.file2list(ROUND_START_MUSIC_LIST, "\n")
 		login_music = pick(music)
 	else
 		login_music = "config/title_music/sounds/[pick(music)]"
-	
+
 
 	if(!GLOB.syndicate_code_phrase)
 		GLOB.syndicate_code_phrase	= generate_code_phrase()
 	if(!GLOB.syndicate_code_response)
 		GLOB.syndicate_code_response = generate_code_phrase()
+
+	generate_selectable_species()
```

Comment:
I assume this proc could be better placed elsewhere

Reply:
it might be needed earlier than this, since i'm 99% sure you already have access to character setup when ticker is still initializing

Reply:
You can access Setup Character once SSJob is done

Reply:
Should I put it in SSjobs setup then?